src/fitting.py

```diff
@@ -167,3 +167,49 @@ def fit_Y_lms_binning_least_squares(M_xyz, L_max, numb_bins, only_even_Ls=False)
     B_lms = np.linalg.solve( ylm_features.T @ ylm_features, ylm_features.T @ angular_hist_flat)
 
     return B_lms, lm_order
+
+def validation_cross_entropy(data_val_xyz, labels, model_params, L_max, only_even_Ls=False):
+    """
+    computes the cross entropy using the shpericial harmonic distribution and labeled data
+    from clustering
+
+    Parameters
+    ------------
+    data_val_xyz: array Nx3
+        Validation data used to compute the cross entropy of the clasification
+    labels: array of ints
+        Each entry varries from 0 to (number classes - 1). Used to reference the model params.
```

Comment:
What happens to data points with '-1' label? This is needed for optics clustering.